contracts/Loans.vy

```diff
@@ -409,7 +399,8 @@ def _reserve(
         msg.sender,
         msg.sender,
         newLoanId,
-        ILendingPoolPeripheral(self.lendingPoolPeripheralContract).erc20TokenContract()
+        ILendingPoolPeripheral(self.lendingPoolPeripheralContract).erc20TokenContract(),
+        _interest * 360 / (_maturity - block.timestamp)
```

Comment:
Since we're adding data to the `LoanCreated` event, which is something I wanted us to do in V2, we could go ahead and also add the loan amount, loan duration and collateral NFTs. What do you think?